content/non-web-ict.md

```diff
@@ -28,7 +28,9 @@ ref: /standards-guidelines/wcag/non-web-ict/
 
 This page introduces the document: "Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)".
 
-Quick link: [WCAG2ICT Note](https://www.w3.org/TR/wcag2ict/) (from 2013)
+Quick links:
+* [WCAG2ICT Note for WCAG 2.o, 2013](https://www.w3.org/TR/wcag2ict/)
```

Comment:
I'm thinking this should say "WCAG 2.0" instead of "WCAG 2.o".

Reply:
yup. can you do a pull request for fixes, additions, edits, etc.

Reply:
@shawna-slh I would have to do a fork of this, as I have no access to edit this branch. Is that what you intend for me to do?

Reply:
Yes please!